Orange/tests/test_asn_data_working.csv

```diff
@@ -0,0 +1,9 @@
+PAGE	DATE	DATE_ISO	DATE_ISO_CLEANED	DECADE	YEAR	MONTH	ID	TYPE	REGISTRATION	OPERATOR	FATALITIES	FATALITIES_CLEANED	LOCATION	CATEGORY
+1	18.08.26	1926-08-18	1926-08-18	1920	1926	8	19260818-0	Blériot 155	F-AIEB	Air Union	4	4	Hurst, near ...	A1
+1	02-OCT-1926	1926-10-02	1926-10-02	1920	1926	10	19261002-0	Blériot 155	F-AICQ	Air Union	7	7	Leigh, Kent	A1
+1	22.08.27	1927-08-22	1927-08-22	1920	1927	8	19270822-0	Fokker F.VIII	H-NADU	KLM	1	1	St. Julians,...	A1
+1	15.02.28	1928-02-15	1928-02-15	1920	1928	2	19280215-0	Handley Page W.8b	G-EBBG	Imperial Airways	0	0	near Abbeville	A1
+1	11-MAR-1928	1928-03-11	1928-03-11	1920	1928	3	19280311-0	Farman F.60 Goliath	F-AEFC	Air Union	2	2	near Folkestone, ...	A1
+1	11.04.28	1928-04-11	1928-04-11	1920	1928	4	19280411-0	Fokker F.VIII	H-NAEE	KLM	0	0	Nigtevecht	A1
+1	12-MAY-1928	1928-05-12	1928-05-12	1920	1928	5	19280512-0	Ford 4-AT-A Tri-Motor	NC1492	Ford Air Freight Lines	2	2	Dearborn, MI	A1
+1	23-MAY-1928	1928-05-23	1928-05-23	1920	1928	5	19280523-0	Farman F.63bis Goliath	F-AEIE	SGTA	3	3	near Köln-Butzwei...	A1
```

Comment:
There's not a single row / column I could remove to still be able to reproduce the issue.